src/components/About/About.module.css

```diff
@@ -1,3 +1,139 @@
 .title {
   text-align: center;
 }
+
+@keyframes ldio {
+    0% { opacity: 1 }
+    100% { opacity: 0 }
+  }
+  .ldio div {
+    left: 94px;
+    top: 48px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5763907)